autorest/azure/auth/auth.go

```diff
@@ -89,10 +89,10 @@ func NewAuthorizerFromEnvironment() (autorest.Authorizer, error) {
 		return config.Authorizer()
 	}
 
-	//4. By default return MSI
-	config := NewMSIConfig()
+	// 4. IDMS
+	config := NewIDMSConfig()
```

Comment:
>NewIDMSConfig [](start = 11, length = 13)

same thing, no need for a breaking change

Reply:
Fixed